tests/pytests/test_parser.py

```diff
@@ -617,3 +617,32 @@ def testModifierList(env):
   }
 }
 '''[1:])
+
+def testTagQueryWithStopwords_V2(env):
+    env = Env(moduleArgs = 'DEFAULT_DIALECT 2')
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 't1', 'TEXT', 't2', 'TAG').ok()
+    env.expect('FT.EXPLAIN', 'idx', '@t2:{as is the with by}').equal(r'''
```

Comment:
Need a test to make sure that with `DIALECT 1` the behavior is as before - stop words are filtered (fix is only in DIALECT 2 or above)